### ISSUE TITLE
rpm package not working on Fedora 35

### ISSUE THREAD

**Issue by Nizzk2** [OpenTabletDriver/OpenTabletDriver#1945]
## Description

I get the following error message when trying to install OpenTabletDriver.rpm version 0.6.0.1 on Fedora 35:
```
$ sudo rpm -i OpenTabletDriver.rpm 
error: Failed dependencies:
	dotnet-runtime-5.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(Microsoft.CSharp) = 4.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(Microsoft.Win32.Primitives) = 4.1.1.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(Microsoft.Win32.Primitives) = 5.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(Microsoft.Win32.Registry) = 4.1.2.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(Newtonsoft.Json) = 12.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System) = 4.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Buffers) = 4.0.2.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Collections) = 4.1.1.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Collections) = 4.1.2.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Collections) = 5.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Collections) = 6.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Collections.Concurrent) = 4.0.14.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Collections.Concurrent) = 6.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Collections.Immutable) = 1.2.5.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Collections.Immutable) = 6.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.ComponentModel) = 6.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.ComponentModel.Annotations) = 4.2.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.ComponentModel.Primitives) = 6.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Console) = 6.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Diagnostics.Debug) = 4.1.1.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Diagnostics.Debug) = 4.1.2.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Diagnostics.Process) = 4.2.1.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Diagnostics.Process) = 6.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Diagnostics.StackTrace) = 4.1.1.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Diagnostics.Tools) = 4.1.1.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Diagnostics.Tools) = 4.1.2.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Diagnostics.TraceSource) = 4.1.1.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Diagnostics.TraceSource) = 4.1.2.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Diagnostics.TraceSource) = 5.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Diagnostics.Tracing) = 4.2.1.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Diagnostics.Tracing) = 6.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.IO.Compression) = 6.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.IO.Compression.ZipFile) = 6.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.IO.FileSystem) = 5.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.IO.Pipes) = 4.1.2.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.IO.Pipes) = 6.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Linq) = 4.2.1.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Linq) = 4.2.2.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Linq) = 5.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Linq) = 6.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Linq.Expressions) = 4.2.1.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Linq.Expressions) = 6.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Linq.Parallel) = 6.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Memory) = 4.0.1.1 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Memory) = 4.1.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Memory) = 4.2.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Memory) = 4.2.1.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Memory) = 6.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Net.Http) = 4.2.1.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Net.Http) = 6.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Net.Primitives) = 4.1.2.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Net.Primitives) = 6.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Net.Sockets) = 4.2.2.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Net.WebSockets) = 4.1.1.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Net.WebSockets) = 4.1.2.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Numerics.Vectors) = 6.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.ObjectModel) = 4.1.1.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.ObjectModel) = 6.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Reflection.Emit) = 4.1.1.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Reflection.Emit.ILGeneration) = 4.0.3.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Reflection.Emit.ILGeneration) = 6.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Reflection.Emit.Lightweight) = 4.0.3.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Reflection.Emit.Lightweight) = 6.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Reflection.Primitives) = 4.1.1.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Reflection.Primitives) = 6.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Resources.ResourceManager) = 4.1.1.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Resources.ResourceManager) = 4.1.2.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Runtime) = 4.2.1.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Runtime) = 4.2.2.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Runtime) = 5.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Runtime) = 6.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Runtime.CompilerServices.Unsafe) = 4.0.4.1 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Runtime.CompilerServices.Unsafe) = 4.0.5.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Runtime.CompilerServices.Unsafe) = 4.0.6.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Runtime.CompilerServices.Unsafe) = 6.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Runtime.Extensions) = 4.2.1.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Runtime.Extensions) = 4.2.2.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Runtime.InteropServices) = 4.2.1.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Runtime.InteropServices) = 4.2.2.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Runtime.InteropServices) = 5.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Runtime.InteropServices) = 6.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Runtime.InteropServices.RuntimeInformation) = 4.0.3.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Runtime.InteropServices.RuntimeInformation) = 6.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Runtime.Loader) = 6.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Runtime.Numerics) = 4.1.1.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Runtime.Serialization.Primitives) = 4.2.1.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Runtime.Serialization.Primitives) = 4.2.2.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Security.Cryptography.Algorithms) = 5.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Security.Cryptography.Algorithms) = 6.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Security.Cryptography.Primitives) = 5.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Security.Cryptography.Primitives) = 6.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Text.Encoding.Extensions) = 4.1.1.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Text.Encoding.Extensions) = 4.1.2.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Text.Json) = 6.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Text.RegularExpressions) = 4.2.1.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Text.RegularExpressions) = 5.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Text.RegularExpressions) = 6.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Threading) = 4.1.1.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Threading) = 4.1.2.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Threading) = 5.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Threading) = 6.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Threading.Overlapped) = 5.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Threading.Overlapped) = 6.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Threading.Tasks) = 4.1.1.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Threading.Tasks) = 4.1.2.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Threading.Tasks.Dataflow) = 4.6.5.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Threading.Thread) = 4.1.1.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Threading.Thread) = 5.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Threading.Thread) = 6.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Threading.ThreadPool) = 4.1.1.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Threading.ThreadPool) = 5.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Threading.ThreadPool) = 6.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Threading.Timer) = 4.1.1.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(System.Xml.XDocument) = 4.1.1.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(mscorlib) = 4.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(netstandard) = 2.0.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	mono(netstandard) = 2.1.0.0 is needed by opentabletdriver-0.6.0.0-1.x86_64
	pkgconfig(libevdev) is needed by opentabletdriver-0.6.0.0-1.x86_64
```

## System Information:

| Name             | Value |
| ---------------- | ----- |
| Operating System         | Fedora 35 workstation
| OpenTabletDriver Version |  0.6.0.1
| Tablet                   | --

**Comment by jamesbt365** [comment]
Make sure you have all the dependencies.

**Comment by hwsmm** [comment]
RPM is currently broken because the author of this project didn't update a dependency list of RPM package, and you need to use your package manager to install RPM because `rpm -i` itself won't solve dependencies. You need to use `sudo dnf install ./OpenTabletDriver.rpm` instead.

**Comment by InfinityGhost** [comment]
Currently getting this when installing myself
```
  Running scriptlet: opentabletdriver-0.6.0.2-1.x86_64                                                                                                                                                          64/64 
/var/tmp/rpm-tmp.LWhnm6: line 1: fg: no job control
error: %prein(opentabletdriver-0.6.0.2-1.x86_64) scriptlet failed, exit status 1

Error in PREIN scriptlet in rpm package opentabletdriver
  Running scriptlet: dconf-0.40.0-3.fc34.x86_64                                                                                                                                                                 64/64 
error: opentabletdriver-0.6.0.2-1.x86_64: install failed
```

**Comment by InfinityGhost** [comment]
This is now fixed in v0.6.0.2
